pub/spec/dependabot/pub/file_updater_spec.rb

```diff
@@ -35,13 +26,22 @@
     end
   end
 
-  after do
-    sample_files.each do |f|
-      package = File.basename(f, ".json")
-      @server.unmount "/api/packages/#{package}"
+  after(:all) do
+    @server.shutdown
+  end
+
+  before(:all) do
+    # Because we do the networking in dependency_services we have to run an
+    # actual web server.
+    dev_null = WEBrick::Log.new("/dev/null", 7)
+    @server = WEBrick::HTTPServer.new({ Port: 0, AccessLog: [], Logger: dev_null })
+    Thread.new do
+      @server.start
```

Comment:
(Subjective) Since I think the goal is more readable tests, I thought it was worth mentioning that personally I find it more readable when `before` setup appears before `after` setup in the file, but maybe that's not true for everyone.